main.cpp

```diff
@@ -80,43 +96,95 @@ bool rentACar(int rentalDuration, int carType = 0)  //function declaration with
 /*
  1)
  */
-
+bool soundPlaying(double volume, int onSwitch=0)
+{
+    ignoreUnused(volume, onSwitch);
+    return {};
+}
 /*
  2)
  */
-
+void getWeather(double temp, bool detectRain, char climateZone)
+{
+    ignoreUnused(temp, detectRain, climateZone);
+
+}
 /*
  3)
  */
-
+void keyboardType(int keys = 88, bool electric = true)
+{
+    ignoreUnused(keys, electric);
+
+}
 /*
  4)
  */
-
+int ageFinder(int yourAge, int currentYear)
+{
+    ignoreUnused(yourAge, currentYear);
+    return {};
+}
 /*
  5)
  */
+
+char restaurantRating(char kitchen, char diningRoom, char bathroom, char failingGrade = 'D')
+{
+    ignoreUnused(kitchen, diningRoom, bathroom, failingGrade);
+    return {};
+}
+
 
 /*
  6)
  */
+
+void radarDectector(double currentSpeed, int speedLimit, int specialRestrictions = 3, bool authorized = true)
+{
+    ignoreUnused(currentSpeed, speedLimit, specialRestrictions, authorized);
+}
 
 /*
  7)
  */
-
+
+double voltageDrop(double current, double resistanceA, double resistanceB, double powerFactor)
+{
+    ignoreUnused(current, resistanceA, resistanceB,powerFactor);
+    return {};
+
+}

```

Comment:
Get in the habit of removing empty lines
they are unnecessary and make your code look sloppy
- [ ] revise